types/adeira__test-utils/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for @adeira/test-utils 0.6
+// Project: https://github.com/adeira/universe/tree/master/src/test-utils
```

Comment:
```suggestion
// Project: https://github.com/adeira/universe/tree/master/src/test-utils#readme
```
minore, just diredctly to readme